Assets/MixedRealityToolkit/_Core/Devices/BaseController.cs

```diff
@@ -23,14 +23,39 @@ public abstract class BaseController : IMixedRealityController
         /// <param name="controllerHandedness"></param>
         /// <param name="inputSource"></param>
         /// <param name="interactions"></param>
-        protected BaseController(TrackingState trackingState, Handedness controllerHandedness, IMixedRealityInputSource inputSource = null, MixedRealityInteractionMapping[] interactions = null)
+        protected BaseController(TrackingState trackingState, Handedness controllerHandedness, IMixedRealityInputSource inputSource, MixedRealityInteractionMapping[] interactions)
         {
             TrackingState = trackingState;
             ControllerHandedness = controllerHandedness;
             InputSource = inputSource;
             Interactions = interactions;
+
+            // TODO: consider if these should be passed in or not
+            IsPositionAvailable = false;
+            PositionAccuracy = TrackingAccuracy.None;
+            IsRotationAvailable = false;
+            RotationAccuracy = TrackingAccuracy.None;
         }
 
+        /// <summary>
+        /// Constructor.
+        /// </summary>
+        /// <param name="trackingState"></param>
+        /// <param name="controllerHandedness"></param>
+        /// <param name="inputSource"></param>
+        protected BaseController(TrackingState trackingState, Handedness controllerHandedness, IMixedRealityInputSource inputSource) :
+            this(trackingState, controllerHandedness, inputSource, null)
```

Comment:
What's with the extra constructors?

Reply:
The intent here was to provide a more obvious indication (especially when reading API docs) that it was a supported scenario to use just these parameters. I DID miss one configuration where it might be possible to have interaction mappings without an inputSource